src/NuGet.Services.Validation.Orchestrator/Configuration/ValidationConfigurationExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NuGet.Services.Validation.Orchestrator
+{
+    internal static class ValidationConfigurationExtensions
+    {
+        private const string NuGetValidationStepContentType = "NuGet";
+
+        public static List<ValidationConfigurationItem> GetClassicValidationConfiguration(this ValidationConfiguration validationConfiguration)
+        {
+            List<ValidationConfigurationItem> nugetList = null;
+            if (validationConfiguration?.ValidationSteps?.TryGetValue(NuGetValidationStepContentType, out nugetList) == true)
```

Comment:
I forget, does this work on .NET Framework?

```suggestion
            if (validationConfiguration?.ValidationSteps?.TryGetValue(NuGetValidationStepContentType, out var nugetList))
```

Reply:
1. The expression produces a value of `bool?` type which cannot be used as condition.
2. Compiler complains about use of uninitialized variable below, so `out var` is also out.